engine/src/main/java/org/hibernate/search/cfg/spi/SearchConfigurationBase.java

```diff
@@ -55,4 +55,9 @@ public InstanceInitializer getInstanceInitializer() {
 	public boolean isIdProvidedImplicit() {
 		return false;
 	}
+
+	@Override
+	public boolean isMultitenancyEnabled() {
+		return true;
```

Comment:
Actually, the only reason was backward compatibility. If someone was extending `SearchConfigurationBase` with its own implementation, I didn't want to change the behavior without notice. I think Infinispan does extend `SearchConfigurationBase`, in particular.

On the other hand, I see no problem regarding `WorkerBuildContextForTest`.

As you changed the default value to false, maybe we should add something to the release notes?